assets/case/address-protection.scss

```diff
@@ -0,0 +1,43 @@
+// Address protection styling for forms
+
+// Hides checkboxes
+[id$='_bringerIsUnderAddressProtection'],
+[id$='_accusedIsUnderAddressProtection'],
+[id='party_form_isUnderAddressProtection'],
+[id='Party_isUnderAddressProtection'] {
```

Comment:
You could use just `[id$="IsUnderAddressProtection"]` here to and reduce the number of selectors.

Reply:
Used `[id$='sUnderAddressProtection']`.